tiledb/sm/consolidator/fragment_consolidator.cc

```diff
@@ -583,8 +583,7 @@ Status FragmentConsolidator::consolidate_internal(
     vac_uri =
         array_for_reads->array_directory().get_vacuum_uri(*new_fragment_uri);
   } catch (std::exception& e) {
-    std::throw_with_nested(
-        std::logic_error("[FragmentConsolidator::consolidate_internal] "));
+    std::throw_with_nested(FragmentConsolidatorException(e.what()));
```

Comment:
If your message is just `e.what()`, there's no reason for `throw_with_nested`. `throw_with_nested` should state the problem with the exception in its own terms, not that of the caught exception.